colors/build.js

```diff
@@ -62,15 +47,7 @@ Name: Firefox/Photon
     'ext': 'gpl'
   },
   'libreoffice': {
-    'output': [`<?xml version="1.0" encoding="UTF-8"?>
-<ooo:color-table
-  xmlns:office="urn:oasis:names:tc:opendocument:xmlns:office:1.0"
-  xmlns:draw="urn:oasis:names:tc:opendocument:xmlns:drawing:1.0"
-  xmlns:xlink="http://www.w3.org/1999/xlink"
-  xmlns:svg="http://www.w3.org/2000/svg"
-  xmlns:ooo="http://openoffice.org/2004/office">
-
-`],
+    'output': [`<?xml version="1.0" encoding="UTF-8"?>\n<ooo:color-table\n  xmlns:office="urn:oasis:names:tc:opendocument:xmlns:office:1.0"\n  xmlns:draw="urn:oasis:names:tc:opendocument:xmlns:drawing:1.0"\n  xmlns:xlink="http://www.w3.org/1999/xlink"\n  xmlns:svg="http://www.w3.org/2000/svg"\n  xmlns:ooo="http://openoffice.org/2004/office">\n\n`],
```

Comment:
I find this much harder to read than the form with in-line newlines. If you think it looks better, you could pull the headers out into their own variables, and reference them here…

Reply:
Oh yeah, i meant that it made the whole file more legible since the inline-newlines and indentation kind of blow up the flow of the document. (https://github.com/johngruen/design-tokens/blob/ac132bd64537852d1c8747d94b42abea3e178793/colors/build.js)

I can extract them though.